android/build.gradle

```diff
@@ -2,11 +2,13 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.3"
-        minSdkVersion = 21
+        minSdkVersion = 24
```

Comment:
Is this really needed?

Reply:
reverted to `21`, but before building detox I will exchange `21` => `24` in circleCI job